src/routes.js

```diff
@@ -1,20 +0,0 @@
-var handler = require("./handlers.js");
-
-function router(req, res) {
-  var url = req.url;
-  if (url === '/' || url.startsWith('/public')) {
-    handler.publicHandler(req, res);
-  } else if (url === '/create-cohort') {
-    handler.createCohortHandler(req, res);
-  } else if (url === '/add-mentor') {
-    handler.createNewMentor(req,res);
-  } else if (url === '/cohorts') {
-    handler.viewCohortsHandler(req, res)
-  } else if (url.startsWith('/weeks')) {
-    handler.viewWeeksHandler(req, res);
-  } else {
-    handler.noPageHandler(req, res);
-  }
```

Comment:
This doesn't take into account the method. Try something like this https://github.com/foundersandcoders/ws-session-management/blob/master/exercise-2/router.js#L10